README.md

```diff
@@ -162,6 +162,17 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+Creating PAM or Native Credentials File (.irodsA)
+-------------------------------------------------
+
+```Two free functions exist for creating encoded authentication files.
```

Comment:
What is the purpose of the statement following the triple backticks here?

Normally, one would specify a programming language abbreviation to get syntax highlighting.

Reply:
Those backticks are misplace; I've now moved them.